src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -391,4 +395,30 @@ public void validateRoles(List<String> rolesToValidate) {
         assertThat(rolesSize.size()).isEqualTo(rolesToValidate.size());
         assertThat(rolesSize).containsAll(rolesToValidate);
     }
+
+    @Test
+    @ToggleEnable(mapKey = "OrganisationInternalController.updateOrgMfaStatus", withFeature = true)
+    public void updateOrgMfaScenario() {
+        setUpTestData();
```

Comment:
yes, because even though it is part of the same class, this is considered as a separate test as we have added the @Test annotation, so it needs to be run again.